src/libraries/System.Text.Json/src/System.Text.Json.csproj

```diff
@@ -16,6 +16,7 @@
 
 The System.Text.Json library is built-in as part of the shared framework in .NET Runtime. The package can be installed when you need to use it in other target frameworks.</PackageDescription>
     <PackageReadmeFile>README.md</PackageReadmeFile>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
```

Comment:
_Moved the comment to the right location._

This is already specified in line 11, although given the fact that during branding the line is removed, we need to make sure that this gets included when we merge to the branch.

You also need to bump the ServicingVersion to 3. Line 12.
```suggestion
```

Reply:
Preferrably lets put it in line 11, so that GeneratePackageOnBuild and ServicingVersion are next to each other.